packages/lit-dev-tools-cjs/src/search/plugin.ts

```diff
@@ -15,16 +15,35 @@ import {PageSearchChunker} from './PageSearchChunker.js';
 type UrlToFile = Map<string, string>;
 
 /**
- * Data to be Indexed.
+ * Generic that describes the type of document.
+ */
+interface DocType<T extends string, U extends string> {
```

Comment:
A little unfortunate this type is duplicated.

Reply:
Agreed. Can't think of a good way to assert that they're both the same since they're separate projects, and importing across projects seems to cause unrelated TS failures. I'm not totally sure that this is too much of an issue since I don't really see us making changes to these types too often

Reply:
I think a comment that mentions these should be in sync would be nice.